### PR TITLE
Razer Blade 2019 advanced support

### PR DESCRIPTION
**Pull request by scramble45**
Add 2019 Advance Razer Blade Support

- [x] Tested and this seems to work

### CODE REVIEW

**Comment by scramble45** [comment]
If anyone has the correct image files for this PR please link them here.

**Comment by scramble45** [comment]
@terrycain - can we get this merged in?

**Comment by Human** [comment]
This is working for me. @scramble45 do no other effects work on this laptop?

**Comment by xkludge** [comment]
I have been using @scramble45 branch for a week and appears to work as well

**Comment by scramble45** [comment]
if anything else needs to be done with this PR, someone let me know so we can get this merged in.

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ The below devices are fully feature supported by OpenRazer which means all avail
 | Razer Blade 15 (2018)                         |  1532:0233  |
 | Razer Blade 15 (2018) Base Model              |  1532:023B  |
 | Razer Blade 15 (2018) Mercury                 |  1532:0240  |
+| Razer Blade 15 (2019) Advanced                |  1532:023A  |
```
Comment (z3ntu) [inline README.md]: Please sort this by the USB IDs (so put this before the "Base Model" above

Comment (z3ntu) [inline README.md]: Please sort this by the USB IDs (so put this before the "Base Model" above

File: daemon/openrazer_daemon/hardware/keyboards.py [hunk 2]
```diff
@@ -560,7 +568,8 @@ class RazerBlackWidowUltimate2016(_RippleKeyboard):
     """
     Class for the Razer BlackWidow Ultimate 2016
     """
-    EVENT_FILE_REGEX = re.compile(r'.*BlackWidow_Ultimate_2016(-if01)?-event-kbd')
+    EVENT_FILE_REGEX = re.compile(
+        r'.*BlackWidow_Ultimate_2016(-if01)?-event-kbd')
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/keyboards.py]: Please remove all the reformats that you didn't touch manually. (You maybe didn't use the max-line-length or whatever option with autopep8?)

File: driver/razerkbd_driver.h [hunk 3]
```diff
@@ -95,6 +95,10 @@
 #define USB_DEVICE_ID_RAZER_BLADE_2018_BASE 0x023b
 #endif
 
+#ifndef USB_DEVICE_ID_RAZER_BLADE_2019_ADV
+#define USB_DEVICE_ID_RAZER_BLADE_2019_ADV 0x023a
+#endif
```
Comment (z3ntu) [inline driver/razerkbd_driver.h:96]: Put these lines above the 2018_BASE lines above (again, ordered by PID)

File: install_files/appstream/io.github.openrazer.openrazer.metainfo.xml [hunk 4]
```diff
@@ -74,6 +74,7 @@
     <modalias>usb:v1532p0232d*</modalias>
     <modalias>usb:v1532p0233d*</modalias>
     <modalias>usb:v1532p023bd*</modalias>
+    <modalias>usb:v1532p023ad*</modalias>
```
Comment (z3ntu) [inline install_files/appstream/io.github.openrazer.openrazer.metainfo.xml:77]: You can use a script in `scripts/` in the repo to generate this file, or , move the line manually one up

File: install_files/udev/99-razer.rules [hunk 5]
```diff
@@ -10,7 +10,7 @@ ATTRS{idProduct}=="0013|0020|0024|0025|002f|0032|0034|0036|0037|0039|0040|0041|0
     ENV{ID_RAZER_CHROMA}="1", ENV{RAZER_DRIVER}="razermouse"
 
 # Keyboards
-ATTRS{idProduct}=="010d|010e|0111|0113|011b|011a|010f|0201|0202|0203|0204|0205|0207|0208|0209|020f|0210|0211|0214|0216|021a|0217|021a|021e|021f|0220|0221|0224|0225|022a|022d|022f|0232|0233|023b|0240", \
+ATTRS{idProduct}=="010d|010e|0111|0113|011b|011a|010f|0201|0202|0203|0204|0205|0207|0208|0209|020f|0210|0211|0214|0216|021a|0217|021a|021e|021f|0220|0221|0224|0225|022a|022d|022f|0232|0233|023b|023a|0240", \
```
Comment (z3ntu) [inline install_files/udev/99-razer.rules]: `023a` should be before `023b` again

**Review by z3ntu — CHANGES_REQUESTED** [review]
See comments, otherwise looks good to me

**Comment by scramble45** [comment]
@z3ntu thanks for the review let me know if I missed anything.

**Review by z3ntu — APPROVED** [review]
LGTM (pushed a commit fixing two little style issues)

**Comment by scramble45** [comment]
@z3ntu thanks next time I will focus on sticking with your styling. I'm new to `autopep8`

**Pull request merged**